src/SortableContainer/index.js

```diff
@@ -207,7 +208,7 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
     handleMove = event => {
       const {distance, pressThreshold} = this.props;
 
-      if (!this.state.sorting && this._touched) {
+      if (!this.state.sorting && this._touched && !this._beforeSortStartIsCalled) {
```

Comment:
We need this check because if the user drag before the promise is resolved, the drag is cancel